spec/system/application/timeout_warning_spec.rb

```diff
@@ -0,0 +1,18 @@
+require "rails_helper"
+
+RSpec.describe "/*", type: :system do
+  context "when user is signed in" do
+    let(:user) { create(:volunteer) }
+    before do
+      sign_in user
+    end
+    it "renders the seconds before logout as a javascript variable" do
+      visit "/"
+      parsed_page = Nokogiri::HTML(page.html)
+      expect(parsed_page.at("script").text.strip).to include(user.timeout_in.in_seconds.to_s)
+    end
+
+    xit "warns the user two mintues before logout", js: true do
+    end
```

Comment:
@ErinClaudio this looks good! Can you just remove this spec on line 15 if we are not going to test for it?

Reply:
you got it and Thank you

Reply:
@littleforest Thank you for the review. When you have a moment please